ecs-init/volumes/ecs_volume_plugin.go

```diff
@@ -201,32 +188,148 @@ func deleteMountPath(path string) error {
 
 // Mount implements Docker volume plugin's Mount Method
 func (a *AmazonECSVolumePlugin) Mount(r *volume.MountRequest) (*volume.MountResponse, error) {
```

Comment:
nit - instead of `a` can we use `p` (for plugin) instead? Feels a little more intuitive that way. Same thing for using `d` for driver

Reply:
Yes. I will make this change.

Reply:
Actually all methods of `AmazonECSVolumePlugin` use `a` variable to reference a pointer to the struct... I think we should change all of them at once but I want to keep this PR's diff cleaner. Is it okay if I do it in another PR as a follow-up?

Reply:
Sounds good